pyteal/ast/abi/address.py

```diff
@@ -56,7 +56,7 @@ def get(self) -> Expr:
 
         The expression will have the type TealType.bytes.
         """
-        return self.stored_value.load()
+        return self._stored_value.load()
```

Comment:
I agree that `stored_value` should have been private, but unfortunately it wasn't. Making it private is a breaking change. I doubt many if any people are actually using it as opposed to the clearly preferable `get()` methods. However, we should decide how to handle this situation deliberately.

@barnjamin @michaeldiamant @jasonpaulos any thoughts?

Reply:
We probably could get away with it given our previous warnings.

Reply:
agree, if we're going to break it we should break it now

Reply:
I suppose the consensus we reached in the live discussion is that, we want to make `stored_value` hidden as `_stored_value`, and probably it is a good time to have breaking change early?